module-4/backend/week_four.md

```diff
@@ -5,15 +5,43 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's your favorite project management tool?
+
+Waffle for its user friendliness and Pivotal Tracker for being detail oriented. 
+
 2. Why do we create staging environments?
+
+So that we can try out different things in different environments that don't break the app. Can change things in staging that won't ruin production. 
+
 3. What are the characteristics of a good README (in your opinion)?
+
+Clear and concise. Has instructions for launching the app that work, and has badges describing the project. 
+
 4. What's one main improvement you're going to make to your code regarding accessibility issues?
+
+Work very hard at making my websites easy to tab through and colorblind friendly. 
+
 5. What are some basic security concerns to be aware of when building applications?
+
+Injection seems to be the big one. Don't let people inject things onto your website and have it be SSL secure. Someone could save a users acts and replicate them later. 
+
 6. Why is continuous integration helpful/important?
+
+This allows us to run all of our tests before pushing to a branch to make sure everything is functioning properly. Makes sure we don't break everything and allows the app to stay up to date. 
```

Comment:
yup - it also allows you to run the application in different environments to test compatibility between different machines for collaboration sake